spec/features/request_a_visit_spec.rb

```diff
@@ -79,7 +79,7 @@
     click_button 'Continue'
 
     within('.date-box-2') do
-      click_button "Change date and time slot"
+      click_button "Change slot 2 date and time"
```

Comment:
I didn't spot the duplicate 'slot'... I meant using the ordinal numbers so:
`Change the second slot date and time`
`Change the first slot date and time`